catalogue/webapp/test/utils/dates.test.ts

```diff
@@ -396,4 +422,15 @@ describe("isRequestableDate: checks the date falls between 2 specified dates and
     });
     expect(result).toEqual(true);
   });
+
+  it('returns true if there are start and end dates and the date falls on the start date', () => {
+    const result = isRequestableDate({
+      date: london('2019-12-17T01:00:00Z'),
+      startDate: london('2019-12-17T12:00:00Z'),
+      endDate: london('2019-12-31'),
+      excludedDates: [],
+      excludedDays: [],
+    });
+    expect(result).toEqual(true);
+  });
```

Comment:
This is defending against a mistake I made in the initial implementation of #8405 – the condition should be `startDate <= date || isSameDay(startDate, date)`, but I'd only implemented the first half of the condition.

It passes here (because it's the correct, existing behaviour), and will hang around until I come back to this code.